hotel-management/src/constants/routePath.ts

```diff
@@ -0,0 +1,12 @@
+const ROUTE_PATH = {
```

Comment:
Rename file, constants from `routePath`, `ROUTE_PATH` to `routes`, `ROUTES` remember the plural